src/PrintAst.ml

```diff
@@ -181,6 +181,8 @@ and print_expr { node; typ } =
   match node with
   | EComment (s, e, s') ->
       surround 2 1 (string s) (print_expr e) (string s')
+  | EStandaloneComment s ->
+      surround 2 1 (string "/*") (string s) (string "*/")
```

Comment:
@tahina-pro you need to use `words` here

Reply:
see http://gallium.inria.fr/~fpottier/pprint/doc/PPrintCombinators.html

Reply:
this printer here is for debugging purposes only